### PR TITLE
Add blog redirection

### PR DESCRIPTION
**Pull request by bschmoker**
:+1: to merge when ready and [content is available](http://stixproject.tumblr.com)

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -2,3 +2,4 @@ source 'http://rubygems.org'
 
 gem 'stix_schema_spy', '>= 1.2.3' # Used for the documentation auto-generation
 gem 'github-pages'             # Used to post to github pages
+gem 'jekyll-redirect-from' # for meta redirects
```
Comment (gtback) [inline Gemfile]: The `github-pages` gem [includes](https://pages.github.com/versions/) the `jekyll-redirect-from` gem. Is this necessary?

**Comment by gtback** [comment]
This looks a bit strange to me when I'm rendering it locally. Is this what you intended?

screen shot 2015-03-17 at 11 22 44 am

**Comment by bschmoker** [comment]
Hmm, I may have gotten the offset wrong, may have had it zoomed differently in testing

**Comment by gtback** [comment]
Only the "Start here" is bold. I pushed an alternate version to https://github.com/STIXProject/stixproject.github.io/tree/add-blog-redirect-alt :

screen shot 2015-03-17 at 3 00 27 pm

**Pull request closed without merging**